### ISSUE TITLE
feat(customization): "lock" a field (value or name) to ensure it remains

### ISSUE THREAD

**Issue by nino-filigran** [LuatixHQ/opencti#8014]
## Use case

Sometimes users are working to ensure their data are formatted in the way they want. And when a connector comes and changes the name of a malware, of a country because an alais is found in the deduplication process, it can be annoying for users since they are not able to find visually this entity in a list.

Therefore, we should be able to "lock" the name of an entity so that if a connector/user/feed... comes with an entity having this name as an alias, the entity is still ingested but the name does not change (and the name of the ingested entity goes in the alias). 
This is a use case of confidence level by attribute.

**Comment by ericWadeFord** [comment]
Another possible method would be to allow users to set a confidence level for each value/name.

**Comment by nino-filigran** [comment]
Yes indeed, it is something that we have thought about: https://github.com/OpenCTI-Platform/opencti/issues/2517

**Comment by jasonkasih** [comment]
Following this issue. Due to the primary name of a Country was changed due to enrichment, it affect the dashboard for the country naming.

**Comment by UncleRaymondo** [comment]
> Following this issue. Due to the primary name of a Country was changed due to enrichment, it affect the dashboard for the country naming.

Yep, I'm now seeing this as well, ingesting a CSV field that has the 2-character country codes, configured to the Country entity, it's overriding the proper names of the countries with the two codes. This, in turn, removes all the flags and affects all the widgets where the country entity is used.

<img width="624" height="512" alt="Image" src="https://github.com/user-attachments/assets/60db5156-29ab-49fd-8d8f-bec19daf1340" />

Super annoying, hoping to see this fixed soon!
